### ISSUE TITLE
Easier deletion of tasks

### ISSUE THREAD

**Issue by zhgwn** [agalwood/Motrix#1594]
**Is your feature request related to a problem? Please describe.**
I'm always frustrated when I have a lot of downloading in my "stopped" tab, and I would like to delete them all, clicking each time on the cross for each one ... And it takes a time it shouldn't.

**Describe the solution you'd like**
I would like to have a faster way to delete those tasks, either by being capable of doing a selection, and then delete, or via using the delete key on the keyboard that would act as clicking on the cross.

**Comment by Cyclics** [comment]
Agree. Really annoying when you want to cancel a bunch of downloading tasks.
